R/drive_mime_type.R

```diff
@@ -105,3 +105,12 @@ one_ext <- function(type) {
   }
   m
 }
+
+drive_show_mime_type <- function(file) {
+  file <- as_dribble(file)
+  file <- confirm_some_files(file)
+  file[["mime_type"]] <- NULL
```

Comment:
If you do promote 2x on the same file, I think it will just add an additional column instead of updating the current one.